Basic-Car-Maintenance/Shared/Utilities/Constants.swift

```diff
@@ -16,3 +16,34 @@ enum FirestoreCollection {
 enum FirestoreField {
     static let userID = "userID"
 }
+
+enum SFSymbol {
+    //Tabview icons
+    static let dashboard = "list.dash.header.rectangle"
+    static let odometer = "gauge"
+    static let settings = "gear"
+
+    //Navigation header icons
+    static let filter = "line.3.horizontal.decrease.circle"
+    static let add = "plus"
+    
+    //Dashboard icons
+    static let trash = "trash"
+    static let edit = "pencil"
+    static let magnifyingglass = "magnifyingglass"
+    
+    //Settings icons
+    static let document = "doc.badge.plus"
+    static let reportbug = "ladybug"
+    static let contributors = "person.3.fill"
+    static let profile = "person"
+    static let appIcon = "apps.iphone"
```

Comment:
```suggestion
    static let iPhoneWithApps = "apps.iphone"
```

I didn't want `appIcon` on this one cause technically it's not an app icon, it's a phone